yorkies.md

```diff
@@ -67,4 +67,12 @@ The BBC has more [good references](https://www.bbcgoodfood.com/howto/guide/top-t
 3. pre-heat the tray so that the fat is really hot. You may not hit the smokepoint of lard or duck fat
 4. Give the batter a good stir before pouring
 5. Fill each tin to 3/4 full
-6. Bake 15-20 minutes until golden brown
\ No newline at end of file
+6. Bake 15-20 minutes until golden brown
+
+
+
+# Cooking Log
+
+**x-05-20:** Attempted "breakfast" yorkies with jam. One came out nicely, the rest leaked into the tin. Wasn't fun to clean.
```

Comment:
ISO 8601 exists for a reason! YYYY-MM-DD plz ; `?`'s if you're not sure.

Reply:
UK Time is better, but fineeeeeeee